crates/ollama/src/ollama.rs

```diff
@@ -90,6 +93,10 @@ impl Model {
     pub fn max_token_count(&self) -> usize {
         self.max_tokens
     }
+
+    pub fn supports_vision(&self) -> Option<bool> {
```

Comment:
Feels like we could just get by without this and access the field directly. If we keep the setter lets return `bool` instead of `Option<bool>` and default to `false`

Reply:
Moved this to directly use the field directly instead of through a getter.